New Theme Inversion

j-mie6 opened the pull request:
Some people were unapply with the sheer in-your-face redness of the new theme introduced in #573, so this just inverts the primary and secondary colours, creating a bit better of a balance. With this, the lighter red is brought back, which is a bit more soft (but had bad contrast at the small sizes on the nav-bar), and the landing page changes from #586 have been incorporated.

However, the landing page in Laika is not particularly well set-up for light backgrounds, so while we wait for that to be properly decoupled (text colour-wise), there is an _**interim**_ darker landing page (note the button colour on that screenshot has since been altered):

![image](https://github.com/typelevel/sbt-typelevel/assets/5148976/2fca0369-ce72-443d-9a2b-e8692d9226f3)

![image](https://github.com/typelevel/sbt-typelevel/assets/5148976/249313ba-6b76-44fe-afee-de397687a34c)

Obviously, there is no default landing page for SBT typelevel currently, but this means the theme is currently a bit more future proofed. Eventually, the landing page will be lightened and all will be well with the world.

j-mie6 commented:
With the merging of #594 the interim dark landing colours have been removed in favour of:

![image](https://github.com/typelevel/sbt-typelevel/assets/5148976/5aa97f5e-2052-4853-ace8-99e6fc4f9a27)

jenshalm commented:
A few fairly minor things I noticed for the normal content pages:

* The `primaryMedium` used for borders and lines is very light and barely visible. If you want to keep this tint for the landing page gradient, you could create one that is a tad darker for the normal pages.

* Literals in syntax highlighting are very dark, I find them hard to read. Even if it's temporary I think it would be good to make it a bit brighter.

* The background for code blocks is similar, but different to `primary`. It _might_ look more harmonic if you use `primary` as syntax-bg, too

j-mie6 commented:
Addressed some of those (kept code background the same, as it is the colour for the TL logo's font and the main text colour:

![image](https://github.com/typelevel/sbt-typelevel/assets/5148976/c04bca4a-bbac-4711-b941-617d20f062c2)

(Also removed some unneeded colours, but kept `redTl` just in case)

jenshalm commented:
Looks good to me now.
